src/translations/pt/coreComponents.js

```diff
@@ -0,0 +1,611 @@
+export default {
+  title: "Componentes principais",
+  description:
+    "Componentes são os principais blocos de construção para interfaces de usuário. A construção de uma biblioteca de componentes reutilizáveis aprimora o fluxo de trabalho de desenvolvimento do produto, reduzindo o design e a dívida tecnológica e acelerando o processo. Os componentes principais não podem ser desmembrados em peças granulares sem perder seu significado.",
```

Comment:
Second sentence could be rewritten as "...reduzindo dívidas técnicas de design e desenvolvimento". In the translation the technical debt part seemed only related to tech. Also, I believe "desenvolvimento", "programação" or even "código" feel more adequate then "tecnológica".